src/features/pagination/js/pagination.js

```diff
@@ -146,12 +176,14 @@
             var visibleRows = renderableRows.filter(function (row) { return row.visible; });
             grid.options.totalItems = visibleRows.length;
 
-            var firstRow = (currentPage - 1) * pageSize;
+            var firstRow = publicApi.methods.pagination.getFirstRowIndex();
+            var lastRow  = publicApi.methods.pagination.getLastRowIndex();
```

Comment:
like so?
```javascript
var firstRow = blahBlah(),
    lastRow  = blahBlah();
```
I don't see that pattern used elsewhere. Can you clarify what you mean?